components/menu/stories/menu.stories.ts

```diff
@@ -0,0 +1,433 @@
+
+import '@aurodesignsystem/auro-button';
+import '@aurodesignsystem/auro-input';
+import '@aurodesignsystem/auro-loader';
+
+AuroMenu.register(); // registering to `auro-menu`
+
+AuroMenu.register('custom-menu');
+
+const meta: Meta = {
+  component: "auro-menu",
+  title: 'Menu',
+};
+export default meta;
+
+type Story = StoryObj;
+
+export const Basic: Story = {
+  render: () => html`
+<auro-menu>
+  <auro-menuoption value="stops">Stops</auro-menuoption>
+  <auro-menuoption value="price">Price</auro-menuoption>
+  <auro-menuoption value="duration">Duration</auro-menuoption>
+  <auro-menuoption value="departure">Departure</auro-menuoption>
+  <auro-menuoption value="arrival">Arrival</auro-menuoption>
+</auro-menu>
+  `
+};
+
+export const NoCheckmark: Story = {
+  render: () => html`
+<auro-menu nocheckmark>
+  <auro-menuoption value="stops">Stops</auro-menuoption>
+  <auro-menuoption value="price">Price</auro-menuoption>
+  <auro-menuoption value="duration">Duration</auro-menuoption>
+  <hr>
+  <auro-menu>
+    <auro-menuoption value="apples">Apples</auro-menuoption>
+    <auro-menuoption value="oranges">Oranges</auro-menuoption>
+    <auro-menuoption value="pears">Pears</auro-menuoption>
+    <auro-menuoption value="grapes">Grapes</auro-menuoption>
+    <auro-menuoption value="kiwi">Kiwi</auro-menuoption>
+    <hr>
+    <auro-menu>
+      <auro-menuoption value="person">Person</auro-menuoption>
+      <auro-menuoption value="woman">Woman</auro-menuoption>
+      <auro-menuoption value="man">Man</auro-menuoption>
+      <auro-menuoption value="camera">Camera</auro-menuoption>
+      <auro-menuoption value="tv">TV</auro-menuoption>
+    </auro-menu>
+  </auro-menu>
+  <hr>
+  <auro-menuoption value="departure">Departure</auro-menuoption>
+  <auro-menuoption value="arrival">Arrival</auro-menuoption>
+  <hr>
+  <auro-menu>
+    <auro-menuoption value="cars">Cars</auro-menuoption>
+    <auro-menuoption value="trucks">Trucks</auro-menuoption>
+    <auro-menuoption value="boats">Boats</auro-menuoption>
+    <auro-menuoption value="planes">Planes</auro-menuoption>
+    <auro-menuoption value="motorcycles">Motorcycles</auro-menuoption>
+  </auro-menu>
+</auro-menu>
+  `
+};
+
+export const DisabledMenu: Story = {
+  render: () => html`
+<auro-menu disabled>
+  <auro-menuoption value="new tab">New tab</auro-menuoption>
+  <auro-menuoption value="new window">New window</auro-menuoption>
+  <auro-menuoption value="open file">Open file</auro-menuoption>
+  <auro-menuoption value="open location">Open location</auro-menuoption>
+  <hr>
+  <auro-menuoption value="close window">Close window</auro-menuoption>
+  <auro-menuoption value="close tab">Close tab</auro-menuoption>
+  <auro-menuoption value="save page as...">Save page as...</auro-menuoption>
+  <hr>
+  <auro-menuoption value="share">Share</auro-menuoption>
+  <hr>
+  <auro-menuoption value="print">Print</auro-menuoption>
+</auro-menu>
+  `
+};
+
+// TODO: This does not function
```

Comment:
The event handler here is quite simple, so I suspect this might be an issue with the component?